helpers/dateTime.js

```diff
@@ -161,6 +161,44 @@ export function getUTCDateTimeFromLocalDateTime(date, time) {
 	return formatDateTimeInISO(utcDateTime);
 }
 
+export function getUTCDateTimeRange(type, diff) {
```

Comment:
I added a new helper since this functionality was used in the new -value component as well as in filter.